app/views/lists/_list.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= div_for(@lists) do |list| %>
```

Comment:
I was thinking about using the collection in the partial rather than passing it the @lists object.  This is good advice. I'll make the change.